courses/advanced-foundry/6-account-abstraction/2-code-overview/+page.md

```diff
@@ -0,0 +1,25 @@
+# Account Abstraction Lesson 2: Code Overview
+
+Welcome to lesson 2 of the Account Abstraction course! In this lesson, we will take a look at the codebase for Ethereum and zkSync. They are minimally structured and serve as a great starting point for learning about account abstraction.
+
+## What You Will Learn
+
+This is just an overview of things to come. You don't need to know much at this point. However, going through the links below can help to give you a glimpse of what is to come in the course. 
```

Comment:
Can we modify 
"You don't need to know much at this point" -> "You don't need to understand fully at this point"